op-program/client/full_run_conf.go

```diff
@@ -0,0 +1,4 @@
+//go:build !smoke_test
+package client
+
+var maximumSteps = -1
```

Comment:
I would recommend leaving a comment about what "-1" means.

Reply:
Yeah, good point. Comment added in https://github.com/ethstorage/optimism/pull/10/commits/84ace42cf260d02eee4a6cc3017a5d1274775520